build/_build/Build.Utilities.cs

```diff
@@ -168,4 +173,91 @@ partial class Build
                 .SetProcessEnvironmentVariables(envVars));
 
         });
+
+    Target GeneratePackageVersions => _ => _
+       .Description("Regenerate the PackageVersions props and .cs files")
+       .Executes(async () =>
+       {
+           var testDir = Solution.GetProject(Projects.ClrProfilerIntegrationTests).Directory;
+
+           var versionGenerator = new PackageVersionGenerator(RootDirectory, testDir);
+           await versionGenerator.GenerateVersions();
+       });
+
+    Target UpdateVendors => _ => _
```

Comment:
Can we rename this to `UpdateVendoredCode`? It updates "code", not "vendors."

Even better, maybe something like `UpdateThirdPartyCode` or `UpdateImportedCode`? I'm not a fan of using "vendor" as a verb.

Reply:
I agree with you that imported/third party code is more descriptive, but currently we use "vendor" in a number of classes and namespaces around this, so I think it makes sense to update everywhere rather than just this target. That will require touching quite a lot of files though, so should probably be done separately (to the backlog!)

I updated this to `UpdateVendoredCode` as a compromise for now.

Reply:
Thanks!